packages/spotlight/src/ui/telemetry/components/insights/envelopes/EnvelopeList.tsx

```diff
@@ -70,5 +71,34 @@ export default function EnvelopeList() {
       </>
     );
   }
-  return <p className="text-primary-300 px-6 py-4">No Envelopes found.</p>;
+  return (
+    <Empty className="min-h-screen">
+      <EmptyHeader>
+        <EmptyTitle>No Envelopes</EmptyTitle>
+        <EmptyDescription>
+          Make sure you have setuped Sentry in your project and enabled Spotlight integration
+        </EmptyDescription>
+      </EmptyHeader>
+      <EmptyContent>
+        <div className="flex gap-4">
+          <a
+            href="https://docs.sentry.io/"
+            target="_blank"
+            rel="noopener noreferrer"
+            className="underline hover:text-primary-400"
+          >
+            Sentry Docs
```

Comment:
i'd be pretty clear here, just one CTA to spotlight docs. we'll redirect you to sentry docs at some point in the spotlight docs if necesary, but having the two CTA could get people to click on sentry docs rather than the spotlight ones.

We should have docs good enough to point you to sentry if necessary.